bindings.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "location_id": "/post_menu",
+    "bindings": [
+      {
+        "label": "Create Zendesk Ticket",
+        "description": "Create ticket in zendesk",
+        "icon": "https://raw.githubusercontent.com/jfrerich/mattermost-applet-zendesk/initial-PR/assets/zendesk.svg",
+        "call": {
+          "url": "https://jasonf.ngrok.io/createform",
```

Comment:
cc @larkox @mickmister @iomodo 3/5 we switch to "path relative to App's RootURL" everywhere in `Call`, `Notify`, etc. All these URLs exist in the proxied namespace. 0/5 if we keep the field name `url` or change `[url_]path`.

Reply:
AWS Lambda functions will not be invoked by sending them HTTP requests directly, we'll be using the AWS `Invoke` API, and will map the AppID/path to the specific Lambda in one of our internal accounts.

Reply:
I am a bit torn between giving liberty to the app developer or making things simpler and more secure.
If we keep the whole URL, I as an app developer can directly route calls to other services/apps/etc... transparently. If not, we would have to have those endpoints configured and then do a redirect or something similar.
But if the URL is always relative to the root, there are less security concerns since everything is sent "to the same place" (even if then that could potentially get redirected somewhere else). Also, we won't need to even consider having another check of allowed URLs for the app (if a call is trying to access an URL not defined in the manifest, it cannot do it).

The big point here is in my opinion, whether we want to allow inter-app calls and how.

Reply:
At this point we should not allow inter-app calls. That should be done by the OAuth in the future. I'd suggest that lambda functions be called only from the apps-plugin or from the cron job. No one else should be able to call lambdas.

Reply:
What about webhooks from third party services? For example, this app expose webhooks for zendesk to communicate with the app.Will those be also proxied by the apps-plugin? How exactly?

Reply:
We need some kind of proxy for 3rd party webhooks as well. For now apps-plugin will act as one. How exactly TBD.